packages/framework-base/src/component-patterns/stagedComponent.tsx

```diff
@@ -0,0 +1,45 @@
+/**
```

Comment:
Any idea why this isn't recognized as a rename?

Reply:
It has the @jsx comment at the top, the types have been moved out of this file, and there is an additional implementation below, which I guess triggered the new logic rather than the rename logic.

Reply:
generally if more than 50% of the file content changes, git won't recognize it as a rename. It acts on content and not on file name.